content/Mobile_Dev/index.adoc

```diff
@@ -0,0 +1,64 @@
+== Multi-Factor Authentication for Mobile Devices
+Protecting user accounts does not stop at desktops and personal computers. Consumers have continually shifted to mobile devices as their primary mechanism for connecting to the web. Many enterprises have already adopted, or have begun to adopt, mobile device management solutions to allow their people to connect to enterprise apps through company or personal devices. Your adaptive multi-factor authentication strategy will be at a huge disadvantage if you do not consider the experience for users who are connecting with their mobile devices. Through this series we are going to explore a variety of different concepts on enabling MFA on mobile devices. By the end of this series you should have the tools, and knowledge on how to select a MFA approach, and how to implement it into your own solution. 
+
+=== Why do mobile devices need specific consideration in authentication strategies?
+As noted above, there has been continued growth in adoption of mobile devices in both the consumer and enterprise space. In this section we are going to outline a few different factors on why this is the right time to take advantage of the needs and requirements of users in different spaces, and merge them with functionality built into mobile devices.
+
+==== Functionality to support MFA built directly into mobile devices
+Mobile devices running iOS and Android have built in functionality to support a variety of different MFA use cases. 
+
+Face ID, Touch ID, and Android biometrics could allow you to tap directly into the platform authenticators built directly into your device. These will help you unlock WebAuthn authentication, which we will explore in more detail momentarily. This piece is also exemplified by the recent announcements from Apple, Google, and Microsoft as they will begin to introduce passkeys (copyable WebAuthn credentials) into their operating systems.
+
+SDK’s and frameworks could also help you to unlock a variety of other MFA flows. For example CryptoTokenKit could help you unlock smart card functionality on Apple devices. Yubico also has SDK’s for both iOS and Android to allow your applications to talk directly to YubiKeys connected to your mobile device.
+
+==== Needs in the consumer space
+The important thing to note in the consumer space is the upward growth in the amount of time that users spend accessing the web on their mobile devices rather than using their PCs. The other goal is to continue to shift consumers away from passwords, into using more secure MFA experiences. Not only does this make user accounts more secure, but can also enable a seamless user experience. There is expected to be a rapid increase in users leveraging WebAuthn credentials due to the introduction of passkeys into popular operating systems.
```

Comment:
* Consider reducing the length of sentences here. Get to the most important point first... 'Consumers demand simple and secure experiences on their mobile devices...'. 
* s/popular operating systems/mobile device ecosystems/

Reply:
You bring up the desire to remove passwords. It may be helpful here to mention the existing technologies and how they fall flat in terms of simplicity and security and present `phishing-resistant credentials` as the path forward. It could be a good segway to passkeys

Reply:
Let me know if the new wording doesn't work for you - may have just created another run on sentence